lib/livebook_web/live/session_live.ex

```diff
@@ -1969,6 +1969,14 @@ defmodule LivebookWeb.SessionLive do
       {:update_smart_cell, _client_id, _cell_id, _cell_state, _delta, _chunks, _reevaluate} ->
```

Comment:
on this file:

> add a clause in `SessionLive` for `handle_action(socket, {:start_evaluation, cell, section})` and do a `push_event` to that cell to clear the decorations.

because

> Theoretically there can be multiple modules with doctests in a cell, so we need to clear the decorations only on new evaluation